### PR TITLE
Implement large file upload test

### PR DESCRIPTION
**Pull request by brastak**
for #152

### CODE REVIEW

File: src/test/java/com/artipie/docker/http/LargeImageITCase.java [hunk 1]
```diff
@@ -0,0 +1,163 @@
...
+     */
+    private Storage storage;
+
+    /**
+     * Repository URL.
+     */
+    private String repo;
+
+    @Test
+    public void largeImageWorks() throws Exception {
```
Comment (olegmoz) [inline src/test/java/com/artipie/docker/http/LargeImageITCase.java:90]: @brastak I'd suggest renaming the test to better reflect that it checks push of large image

File: src/test/java/com/artipie/docker/http/LargeImageITCase.java [hunk 2]
```diff
@@ -0,0 +1,163 @@
...
+                "image", "inspect", "--format={{ index .RepoDigests 0 }}", image
+            );
+            final Digest digest = new Digest.FromString(
+                repodigest
+                    .replace(String.format("%s@", image), "").split("\\R")[0]
+            );
+            MatcherAssert.assertThat(
+                "Could not find blob",
+                new AstoBlobs(this.storage).blob(digest).toCompletableFuture().get().isPresent()
+            );
```
Comment (olegmoz) [inline src/test/java/com/artipie/docker/http/LargeImageITCase.java:112]: @brastak this is quite complicated code for checking that push was successful and it operates on lower abstraction level fit for unit tests, not integration tests. May we simplify it to just check that `docker push` command output stated success?

File: src/test/java/com/artipie/docker/http/LargeImageITCase.java (hunk 1 above)
Comment (brastak) [reply src/test/java/com/artipie/docker/http/LargeImageITCase.java]: @olegmoz renamed to `largeImageUploadWorks`

File: src/test/java/com/artipie/docker/http/LargeImageITCase.java (hunk 2 above)
Comment (brastak) [reply src/test/java/com/artipie/docker/http/LargeImageITCase.java]: @olegmoz fixed

**Comment by brastak** [comment]
@olegmoz Please, see update

File: src/test/java/com/artipie/docker/http/LargeImageITCase.java [hunk 3]
```diff
@@ -0,0 +1,146 @@
...
+            Objects.requireNonNull(
+                Thread.currentThread().getContextClassLoader()
+                    .getResource("large-image/Dockerfile")
+            ).toURI()
+        );
+        Files.copy(dockerfile, this.temp.resolve("Dockerfile"));
+        final String image = String.format("%s/%s", this.repo, LargeImageITCase.IMAGE);
+        try {
+            this.run("build", ".", "-t", image);
+            this.run("push", image);
```
Comment (olegmoz) [inline src/test/java/com/artipie/docker/http/LargeImageITCase.java:92]: @brastak could you please check output of `push` command, the same way it is done in `DockerSliceITCase`?

Comment (olegmoz) [reply src/test/java/com/artipie/docker/http/LargeImageITCase.java:92]: @brastak ping

Comment (brastak) [reply src/test/java/com/artipie/docker/http/LargeImageITCase.java]: @olegmoz Added check. I have waited for #164 be merged

**Comment by olegmoz** [comment]
@brastak thank you, merging it

**Pull request merged**